lib/internal/streams/add-abort-signal.js

```diff
@@ -9,7 +9,7 @@ const eos = require('internal/streams/end-of-stream');
 const { ERR_INVALID_ARG_TYPE } = codes;
 
 // This method is inlined here for readable-stream
-// It also does not allow for signal to not exist on the steam
+// It also does not allow for signal to not exist on the stream
```

Comment:
Unrelated change, but good catch ;) it might be worth having it land in a separate PR (in case this one was reverted or didn't get backported to LTS release lines).